api/admin/controller.py

```diff
@@ -168,6 +170,24 @@ def details(self, data_source, identifier):
         return entry_response(
             AcquisitionFeed.single_entry(self._db, work, annotator)
         )
+        
+    def complaints(self, data_source, identifier):
+        """Return detailed complaint information for admins."""
+        
+        pool = self.load_licensepool(data_source, identifier)
+        if isinstance(pool, ProblemDetail):
+            return pool
+        work = pool.work
+        counter = self._count_complaints_for_work(work)
+        response = dict({
+            "book": { 
```

Comment:
how about "work"? "book" felt awkward when i wrote it, but that's what we use in circulation-web, which might be a problem too.